### ISSUE TITLE
Parallel build fails due to the usage of same container name for the ubuntu container

### ISSUE THREAD

**Issue by prakhassh** [blackducksoftware/hub-docker#23]
is there a way to override the container name blackduck-imageinspector-ubuntu_25 inside the 
utility-scanner-blackduck-docker

**Comment by stevebillings** [comment]
Yes, if you are willing to start the image inspector services yourself (rather than relying on Docker Inspector to start them automatically). More info here: https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/760021011/Docker+Inspector+Deployment+Options

But this page specifically addresses concurrency: https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/760381459/Concurrent+Image+Inspection
